Ladybird/EventLoopPluginQt.cpp

```diff
@@ -18,13 +18,13 @@ namespace Ladybird {
 EventLoopPluginQt::EventLoopPluginQt() = default;
 EventLoopPluginQt::~EventLoopPluginQt() = default;
 
-void EventLoopPluginQt::spin_until(Function<bool()> goal_condition)
+void EventLoopPluginQt::spin_until(JS::SafeFunction<bool()> goal_condition)
```

Comment:
For this commit:

- https://github.com/SerenityOS/serenity/blob/b4f78760f2ad20d7d704ac75ef654d95dfbc64eb/Userland/Libraries/LibWeb/Fetch/Fetching/PendingResponse.cpp#L59 no longer needs to create a handle
- You forgot to update https://github.com/SerenityOS/serenity/blob/b4f78760f2ad20d7d704ac75ef654d95dfbc64eb/Userland/Libraries/LibWeb/Platform/EventLoopPluginSerenity.cpp#L18-L21

Reply:
1) Thanks!
2) I'm confused. That was changed to SafeFunction in that commit I think? Though I just pushed a rebase so maybe it was wrong before and I didn't notice.

Reply:
hmm, now I'm also not sure - maybe Iinked to the wrong place? I'm somewhat sure I saw an instance of plain `Function`, but maybe that was a brain fart...

Reply:
Oh I have found a place that calls it with a regular function. I'll see if I can find any others and update them.

Reply:
yep, `WebEngineCustomData::spin_event_loop_until` was the one!